components/home/landing/Landing.jsx

```diff
@@ -0,0 +1,64 @@
+import {
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  Image,
+  FlatList,
+} from "react-native";
+import { useRouter } from "expo-router";
+
+import styles from "./Landing.style";
+
+const launchTypes = ["Satellite", "Spaceship", "ISS"];
+
+const Landing = ({ searchTerm, setSearchTerm, handleClick }) => {
```

Comment:
we've generally chosen to use `function Langing() {` instead of using an arrow function. Pretty much everything in [this article](https://www.makeuseof.com/javascript-arrow-functions-careful/) are the reasons, although we do typically return an arrow function when a function is returning another function because of clarity